design/cli-install-changes.md

```diff
@@ -0,0 +1,426 @@
+
+      create                                              NAME [flags]
+        --default                                         Sets these new locations to be the new default snapshot locations. Default is false. (NEW)
+        --config  mapStringString                         configuration to use for creating a volume snapshot location. Format is key1=value1,key2=value2 (was also in `velero install --`snapshot-location-config`). Required.
+        --provider string                                 provider name for volume storage. Required.
+        --label-columns stringArray                       a comma-separated list of labels to be displayed as columns
+        --labels mapStringString                          labels to apply to the volume snapshot location
+        --provider string                                 name of the volume snapshot provider (e.g. aws, azure, gcp)
+        --show-labels                                     show labels in the last column
+        --credentials mapStringString                     sets the list of name of the corresponding credentials secret for providers. Format is (provider1:credentials-secret-name1,provider2:credentials-secret-name2,...) (NEW)
+
+      get                                                 Display snapshot locations
+        --default                                         list of unique volume providers and default volume snapshot location (provider1:location-01,provider2:location-02,...) (NEW -- was `server --default-volume-snapshot-locations`))
+```
+
+4) `velero plugin`
+Configuration for plugins.
+
+```
+      add stringArray                           IMAGES [flags] - add plugin container images to install into the Velero Deployment
+
+      get                                       get information for all plugins on the velero server (was `get`)
+        --timeout duration                      maximum time to wait for plugin information to be reported (default 5s)
+
+      remove                                    Remove a plugin [NAME | IMAGE] 
+
+      set
+        --credentials-file mapStringString      configuration to use for creating a secret containing the AIM credentials for a plugin provider. Format is provider:path-to-file. (was `secret-file`)
+        --no-secret                             flag indicating if a secret should be created. Must be used as confirmation if create --secret-file is not provided. Optional. (MOVED FROM install -- not sure we need it?)
+        --sa-annotations mapStringString        annotations to add to the Velero ServiceAccount for GKE. Add iam.gke.io/gcp-service-account=[GSA_NAME]@[PROJECT_NAME].iam.gserviceaccount.com for workload identity. Optional. Format is key1=value1,key2=value2
+        --cacert-file mapStringString           configuration to use for creating a secret containing a custom certificate for an S3 location of a plugin provider. Format is provider:path-to-file. (NEW)
```

Comment:
I think we just want the `--cacert-file` flag on `velero storage-location create/set`, since this applies on a per-BSL basis.

Reply:
Ahhh. Will move.